lib/plugins/boluswizardpreview.js

```diff
@@ -100,6 +103,19 @@ function init() {
       , {label: 'Expected effect', value: '-' + results.effectDisplay + ' ' + sbx.units}
       , {label: 'Expected outcome', value: results.outcomeDisplay + ' ' + sbx.units}
     ];
+    
+    if (results.tempBasalAdjustment) {
+      if (results.tempBasalAdjustment.thirtymin > 0) {
+        info.push( {label: '30m temp basal', value: results.tempBasalAdjustment.thirtymin + '%'}); 
```

Comment:
for some pumps like our animas ping, temp basals are set with +/-%, so 50% = -50%, 110% = +10% and can only be set in 10% increments, so I'd need to add some animas specific display.  Similar to the medtronic specific rounding.  Wonder where that should go.  Guess we'll figure it out later.